research/gaia/gremlin/gremlin_core/src/process/traversal/step/by_key.rs

```diff
@@ -64,10 +62,11 @@ impl FromPb<pb::ByKey> for ByStepOption {
     {
         match by_key_pb.item {
             Some(pb::by_key::Item::Key(key)) => Ok(ByStepOption::OptToken(Token::from_pb(key)?)),
+            // TODO: should be prop_name or prop_id
             Some(pb::by_key::Item::Name(properties_pb)) => {
                 let mut properties = vec![];
-                for prop in properties_pb.item {
-                    properties.push(prop);
+                for prop_name in properties_pb.item {
```

Comment:
How about prop_id ?

Reply:
Since we have not defined prop_id in proto yet, I just add a TODO here and in proto file

Reply:
> How about prop_id ?

Since we have not defined prop_id in proto yet, I just add a TODO here and in proto file